js/src/forum/index.js

```diff
@@ -30,9 +28,13 @@ app.initializers.add('flarum-markdown', function(app) {
   extend(TextEditor.prototype, 'configTextarea', function(value, element, isInitialized, context) {
     if (isInitialized) return;
 
-    const editor = new MarkdownArea(element);
-    editor.disableInline();
-    editor.ignoreTab();
+    const editor = new MarkdownArea(element, {
+      keyMap: {
+        indent: ['Ctrl+m'],
+        outdent: ['Ctrl+M'],
+        inline: ['"', "'", '`']
+      }
+    });
```

Comment:
This should not be part of this PR.

Reply:
Oh, I see you already have a PR for it @ https://github.com/flarum/markdown/pull/20.

Reply:
Not sure how that got in there... Reverted